server/documents/introduction/getting-started.html.eco

```diff
@@ -91,8 +91,8 @@ type        : 'Main'
     <div class="code" data-type="html">
       <!-- You MUST include jQuery before Fomantic -->
       <script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
-      <link rel="stylesheet" type="text/css" href="/semantic/dist/semantic.min.css">
-      <script src="/semantic/dist/semantic.min.js"></script>
+      <link rel="stylesheet" type="text/css" href="semantic/dist/semantic.min.css">
+      <script src="semantic/dist/semantic.min.js"></script>
```

Comment:
This will still be forcing the browser to send the requests and receive 404 responses. The `link` and `script` tag should be encoded format. Can you please modify again as below?
```
&lt;link rel="stylesheet" type="text/css" href="semantic/dist/semantic.min.css"&gt;
&lt;script src="semantic/dist/semantic.min.js"&gt;&lt;/script&gt;
```

Reply:
Sorry. I was wrong. The code examples are already encoded by the script. The possible fix should be removing `semantic` from the path.
```
<link rel="stylesheet" type="text/css" href="/dist/semantic.min.css">
<script src="/dist/semantic.min.js"></script>
```